src/core_modules/capture-core/components/FormFields/Options/SingleSelectBoxes/SingleSelectBoxes.component.js

```diff
@@ -7,29 +7,19 @@ import FormLabel from '@material-ui/core/FormLabel';
 import FormGroup from '@material-ui/core/FormGroup';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import { withStyles } from '@material-ui/core/styles';
-
-import RadioOffIcon from '@material-ui/icons/PanoramaFishEye';
-import RadioOnIcon from '@material-ui/icons/CheckCircle';
-
+import { SingleSelectionCheckedIcon, SingleSelectionUncheckedIcon } from 'capture-ui/Icons';
 import { singleOrientations } from './singleSelectBoxes.const';
+import type { Props } from './singleSelectBoxes.types';
 
-const styles = theme => ({
-    label: theme.typography.formFieldTitle,
-});
-
-type Props = {
-    onBlur: (value: any) => void,
-    options: Array<{text: string, value: any}>,
-    label?: string,
-    nullable?: boolean,
-    value?: any,
-    orientation?: ?$Values<typeof singleOrientations>,
-    required?: ?boolean,
-    classes: {
-        label: string,
+const styles = ({ typography, palette }) => ({
+    label: typography.formFieldTitle,
+    iconSelected: {
+        fill: palette.secondary.main,
     },
-    style?: ?Object,
-};
+    iconDeselected: {
+        fill: palette.grey[700],
```

Comment:
Not really a blocker but would be cool if you can have a look. I think we could use some variation of grey which comes from the `@dhis2/ui` since we want slowly to converge into using that lib. 

You could use it as such `colors.grey700`

Here is the colors provided to us https://github.com/dhis2/ui/blob/master/packages/constants/src/colors.js#L69

Reply:
Thanks, agree, changing.